gui-apps/nwg-panel/nwg-panel-0.4.3.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
```

Comment:
Change it to just `python3_9`, there is no need to support old versions now that gentoo has change default to 3.9.

That way we don't have to revisit later just to remove the older ones.